### ISSUE TITLE
Feature request: direct set/get Buffer type data

### ISSUE THREAD

**Issue by hwen** [zhangyuang/sharedMemory#3]
Hi, in some situation that with large buffer data such as video buffer, it is better to direct set/get Buffer type data.

**Comment by hwen** [comment]
Is Weird that getString spent high cpu and memory
![image](https://github.com/user-attachments/assets/302e372f-c130-42ef-8b4b-10a8d058c303)

**Comment by zhangyuang** [comment]
It's possible, to support it in the future

**Comment by zhangyuang** [comment]
Support share buffer data in 1.0.8

```
// main.js
import { fork } from 'child_process'
import { init, setString, getString, clear, setBuffer, getBuffer } from 'memoryshare'

const memId = "string.link"
const bufferMemId = "buffer.link"
clear(memId)
clear(bufferMemId)

init(memId, 4096) // init share memory block with max size,each memId should be called only once
init(bufferMemId, 4096)


function generateBigString() {
  let bigStr = '';
  for (let i = 0; i < 1; i++) {
    bigStr += 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. ';
  }
  return bigStr;
}


setString(memId, generateBigString())
setBuffer(bufferMemId, Buffer.from(generateBigString()))
fork('./child')

// child.js
const memId = "string.link"
const data = getString(memId)
const bufferData = getBuffer(bufferMemId)
```

**Comment by zhangyuang** [comment]
Benchmark
```
$ yarn bench
Running "data transfer" suite...
Progress: 100%

  ipc:
    26 ops/s, ±96.96%       | slowest, 99.93% slower

  sharememory:
    13 686 ops/s, ±72.23%   | 65.29% slower

  sharebuffer:
    39 428 ops/s, ±8.67%   | fastest

Finished 3 cases!
  Fastest: sharebuffer
  Slowest: ipc
✨  Done in 28.89s.
```

**Comment by hwen** [comment]
seen latest version not publish success that do not have a index.js file
![image](https://github.com/user-attachments/assets/80873968-1652-4507-8cd0-8f7094dd7889)

**Comment by zhangyuang** [comment]
Be fixed in 1.0.9

**Comment by hwen** [comment]
everything is great!
